src/main/java/io/fabric8/elasticsearch/plugin/kibana/KibanaSeed.java

```diff
@@ -96,11 +96,18 @@ public void setDashboards(final OpenshiftRequestContext context, Client client,
         // compare results to projects; handle any deltas (create, delete?)
 
         Set<String> indexPatterns = getProjectNamesFromIndexes(context, client, projectPrefix);
-        LOGGER.debug("Found '{}' Index patterns for user", indexPatterns.size());
+        LOGGER.debug("Found '{}' Index patterns for user {}", indexPatterns.size(), indexPatterns);
```

Comment:
@richm  @ewolinetz any thoughts if this is a security concern dumping visible indices to the logs?